webhook_notification_test.go

```diff
@@ -2,9 +2,35 @@ package braintree
 
 import (
 	"encoding/base64"
+	"io/ioutil"
+	"net/http"
+	"strings"
 	"testing"
 )
 
+func TestWebhookParseRequest(t *testing.T) {
+	t.Parallel()
+
+	gateway := New(Sandbox, "mid", "sz9g7zhxz8838v7h", "0c809a2d2e8f4e4c817900ff441c9554")
```

Comment:
I don't review PRs in this repo very often, so pardon if this a dumb q, but the other tests in this file use a `testGateway` which is constructed with env vars. Any reason to hardcode credentials here?